frontend/views/containers/chatroom/chat-mentions/RenderMessageText.vue

```diff
@@ -0,0 +1,236 @@
+<template lang='pug'>
```

Comment:
some of the existing mention related logics(e.g. splitting the text by possible mentions, using `ProfileCard.vue` in certain cases etc.) are DRYed into this component now, as this needs to be used in multiple places.